starter_ai_agents/adk_email_agent/agent.py

```
@@ -0,0 +1,55 @@
+from dotenv import load_dotenv
+import os
+
+# Load keys
+load_dotenv()
+import os
```

Ruff (0.11.9)

Redefinition of unused `os` from line 2

Remove definition: `os`

(F811)